Makefile

```diff
@@ -61,5 +61,5 @@ validate-dot: $(SNAPSHOTS_OUT)/*.dot
 validate-wgsl: $(SNAPSHOTS_OUT)/*.wgsl
 	@set -e && for file in $^ ; do \
 		echo "Validating" $${file#"$(SNAPSHOTS_OUT)/"};	\
-		cargo run --bin convert --features wgsl-in $${file} >/dev/null; \
+		cargo run --bin convert --features wgsl-in $${file} -- --validation-only; \
```

Comment:
I think we need to approach this differently. It should always be validating, but we don't need to print the contents out. Instead, we can have a special target name, say "-", which means the module will be printed without any files written. That would allow us to avoid the flag.

Reply:
We need validation info for `wgsl-out` shaders, but without printing module to stdout. Maybe we should find a better name for this flag.

Reply:
I addressed this in https://github.com/gfx-rs/naga/pull/744